apps/07jquery-tic-tac-toe/script.js

```diff
@@ -2,4 +2,49 @@
 
 $(document).on('ready', function() {
   // Put app logic in here
+  var playerTurn = 'X';
+  var play = true;
+
+  function stopGame() {
+    $('div[data-cell]:empty').html('<span></span>');
+    play = false;
+  }
+
+  function checkForWin() {
+    // Check for horizontal win
+    if (($("div[data-cell='0']").text() === playerTurn && $("div[data-cell='1']").text() === playerTurn && $("div[data-cell='2']").text() === playerTurn) || ($("div[data-cell='3']").text() === playerTurn && $("div[data-cell='4']").text() === playerTurn && $("div[data-cell='5']").text() === playerTurn) || ($("div[data-cell='6']").text() === playerTurn && $("div[data-cell='7']").text() === playerTurn && $("div[data-cell='8']").text() === playerTurn)) {
```

Comment:
Instead of going through each html item one by one could you use your knowledge of `for loops` and arrays (ex: whitePositions in checkers app) to simplify your checkForWin()??

Hint: `$('[data-cell="'+ winningPositions[i][0] +'"]').text() === playerTurn &&. ...`